libs-scala/resources/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+# Copyright (c) 2020 The DAML Authors. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//bazel_tools:scala.bzl", "da_scala_library", "da_scala_test_suite")
+
+da_scala_library(
+    name = "resources",
+    srcs = glob(["src/main/scala/**/*.scala"]),
+    tags = ["maven_coordinates=com.digitalasset:resources:__VERSION__"],
+    visibility = [
+        "//visibility:public",
+    ],
+    deps = [
+        "@maven//:com_typesafe_akka_akka_actor_2_12",
```

Comment:
Feel free to address this as a separate issue, but I would say it would be great if we could split `resources-akka` as a separate library.

Reply:
Eh, it's a good idea. I can do it here.